transcription/transcription.go

```diff
@@ -29,3 +32,33 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// DownloadFileFromURL downloads an mp3 file locally from a url.
+func DownloadFileFromURL(url string) error {
+	// https://github.com/thbar/golang-playground/blob/master/download-files.go
+	tokens := strings.Split(url, "/")
+	fileName := tokens[len(tokens)-1]
+
+	// TODO: check file existence first with io.IsExist
+	// Create the file
+	output, err := os.Create(fileName)
+	if err != nil {
+		return err
+	}
+	defer output.Close()
+
+	// Get the data
```

Comment:
Why is this a useless comment (esp considering the other comments) - in this piece of code we are actually extracting the file data whereas the other step are creating an output, copying over, etc - ? I'm not sure what the distinction is between useful useless in this case. Maybe a rephrasing? "Get file contents" as opposed to get the data?